packages/fxa-customs-server/lib/user_defined_rules.js

```diff
@@ -60,5 +55,23 @@ module.exports = function (config, fetchRecord, setRecords) {
     }
 
     return result;
+  }
+
+  async function checkUserDefinedRateLimitRulesByIpEmail(
```

Comment:
Tiny nit, we pass in email, then ip, so maybe it should called ByEmailIp or maybe email and ip params should be swapped.